### ISSUE TITLE
Script error when including multiple ineractions in the same js file

### ISSUE THREAD

**Issue by effeappbw** [samuelOsborne/School-Of-Lottie#1]
Hi Samuel--
1) thanks for the tutorials--your content is helpful beyond words.
2) I've hit a snag that I can't figure out (limited js knowledge here)--maybe you can help:

Everything works just great _IF_ i remove all unused scripts from the js file. Moreover, it lets me run #1 with #2 below, but not in reverse order (i.e. copying the bluprint script & placing it above the lighthouse script).

Is it possible that lottie-interactivity.js is somehow creating a scenario whereby one event handler in lottie-interactivity.is is listening for conflicting events in the interactions.js file?

Thanks in advance--

**Comment by samuelOsborne** [comment]
Hi @effeappbw, I don't understand exactly what your problem is, which lesson are you having a problem with? 

And could you please show me the interactions you have from the interactions js file

**Comment by effeappbw** [comment]
Apologies--
I'm referring specifically to the lighthouse  & blueprint sections of your interactivity course.  I'm working directly from your the  interactions.js file found here: 

https://github.com/samuelOsborne/School-Of-Lottie/blob/50a7d1226cea4d54323f3f592508233b53422505/Interactivity-course/lesson-07%20-%20Sync%20to%20cursor/js/interactions.js.

LottieInteractivity.create({
	mode:"scroll",
	player:'#lottie-lighthouse',
	actions: [
		{
			visibility:[0,1],
			type: "seek",
			frames: [0, 300],
		},
	]
});

LottieInteractivity.create({
	mode:"cursor",
	player: "#blueprint-lottie",
	actions: [
		{
			position: { x: [0, 1], y: [0, 1] },
			type: "seek",
			frames: [0, 42]
		}
	]
});

Both work perfectly fine--but only if I delete the unused script from the file.


UPDATE: I think I figured out a solution. I wrapped the scripts with "document.addEventListener("DOMContentLoaded", function () {}). Thanks again for the response!
